core/src/test/java/edu/wpi/grip/util/GRIPCoreTestModule.java

```diff
@@ -69,6 +74,12 @@ public void tearDown() {
     protected void configure() {
         assert setUp : "The GRIPCoreTestModule handler was not set up. Call 'setUp' before passing the injector";
         super.configure();
+        // HTTP server injection bindings
+        bind(GripServer.HttpServerFactory.class).to(GripServerTest.TestServerFactory.class);
+        bind(GripServer.class).asEagerSingleton();
+        bind(MapNetworkPublisherFactory.class)
+                .annotatedWith(Names.named("httpManager"))
+                .to(HttpManager.class);
```

Comment:
Right, but do you need to use dependency injection in your tests?
If you want to test HTTP messages in a test create a concrete instance.
You are writing unit tests not integration tests.